welcome/index.adoc

```diff
@@ -81,9 +81,9 @@ status, connect and group components,
 and modify your code base.
 
 - ** xref:../cli_reference/openshift_developer_cli/understanding-odo.adoc#understanding-odo[Use the developer CLI tool (odo)]**:
-The odo CLI tool lets developers create single- or multi-component applications
+The odo CLI tool lets developers create single or multi-component applications
 easily and automates deployment, build, and service route configurations.
-It abstracts away complex Kubernetes and {product-title} concepts enabling developers to focus on developing their applications.
+It abstracts complex Kubernetes and {product-title} concepts thus allowing developers to focus on developing their applications.
```

Comment:
s/concepts thus allowing/concepts, allowing
"thus" does not seem necessary here